### PR TITLE
Fixes #32009 - removes flot_pie_chart

### PR DESCRIPTION
**Pull request by ezr-ondrej**
This has been deprecated in 4ff6f0a1f0b8c6227f91e6b2bcd5c33e00b22a18 and
had a better solution in React for quite a long time now. We don't want
to support flot charts anymore and this one is first to go.

BTW I had so much fun deleting stuff, I wonder why we don't do it more often :thinking:

### CODE REVIEW

**Comment by tbrisker** [comment]
[test katello] doesn't look related, but just to make sure

File: app/assets/javascripts/charts.js [hunk 1]
```diff
@@ -1,150 +1,3 @@
...
-        hoverable: true,
-        clickable: true,
-      },
-    });
-    bind_hover_event(target, function(item) {
-      var percent = Math.round(item.series.percent);
-      return _.escape(item.series.label) + ' (' + percent + '%)';
-    });
-    $(target).bind('plotclick', function(event, pos, item) {
-      search_on_click(event, item);
```
Comment (tbrisker) [inline app/assets/javascripts/charts.js:62]: You can also drop this function

**Review by tbrisker — CHANGES_REQUESTED** [review]
missed one function, otherwise 💯

File: app/assets/javascripts/charts.js (hunk 1 above)
Comment (ezr-ondrej) [reply app/assets/javascripts/charts.js:62]: Dropped :boom:

**Review by tbrisker — APPROVED** [review]
Good riddance, thanks @ezr-ondrej !

**Pull request merged**